### PR TITLE
tools: script to talk to the staging servers

### PR DESCRIPTION
**Pull request by sergiusens**
This is a sourceable script which would make it easy to work
against the staging servers.

LP: #1630083

Signed-off-by: Sergio Schvezov <sergio.schvezov@ubuntu.com>

### CODE REVIEW

**Comment by kyrofa** [comment]
Though I wonder if it would be even cooler if we made this a subshell, then one can simply exit out of it. That seems to be how other similar tools work.

**Comment by sergiusens** [comment]
@kyrofa I was going for a python venv xp, android does it this way too now that I think of it, I don't mind either way.

**Comment by kyrofa** [comment]
No strong opinions here either, +1 either way. Curious to see what the others think.

**Comment by sergiusens** [comment]
@kyrofa I don't mind, this is my idea of a *vacation* from a bigger refactor going on :-P

File: HACKING.md [hunk 1]
```diff
@@ -49,14 +49,13 @@ Snapcraft has the ability to upload snaps for publication in the Snappy Store.
...
+[staging server](https://login.staging.ubuntu.com), then run:
 
-    UBUNTU_STORE_API_ROOT_URL='https://myapps.developer.staging.ubuntu.com/dev/api/'
-    UBUNTU_STORE_SEARCH_ROOT_URL=https://search.apps.staging.ubuntu.com/
-    UBUNTU_STORE_UPLOAD_ROOT_URL='https://upload.apps.staging.ubuntu.com/'
-    UBUNTU_SSO_API_ROOT_URL='https://login.staging.ubuntu.com/api/v2/'
+    source tools/staging_env.sh
 
+You will see a prompt indicating that you are going to be talking to the staging
+server. Once you are done with working with the staging servers you can run
```
Comment (elopio) [inline HACKING.md]: You should remove one "with":
Once you are done _ working with the staging...

**Pull request merged**